### PR TITLE
Write README

### PR DESCRIPTION
**Pull request by dtan4**
## WHAT

Fill `SUPPORTED TAGS` and `HOW TO USE`, paste Quay.io status badge in README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,44 @@
...
-* TBD
+* `latest`
+ * Terraform 0.4.2
 
 ## HOW TO USE
 
-TBD
+```bash
+# Get the Docker image
+$ docker pull quay.io/wantedly/terraform
```
Comment (spesnova) [inline README.md]: `quay.io/wantedly/terraform:latest` is better :point_up: 
(And I know I do the same thing in other repo...)

## WHY

As far as I know, If we don't specify tag, docker pulls all image (means all tagged images) on the repo.
Sometime this takes a long time.

Comment (dtan4) [inline README.md]: Good catch :+1: thanks!

**Comment by dtan4** [comment]
@spesnova Could you review again & merge this?

**Comment by spesnova** [comment]
:+1:

**Pull request merged**